x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/rule_alerts_summary.tsx

```diff
@@ -28,8 +29,8 @@ export const RuleAlertsSummary = ({
     isLoadingRuleAlertsAggs,
     errorRuleAlertsAggs,
   } = useLoadRuleAlertsAggs({
-    ruleId: rule.id,
```

Comment:
💭 I deleted passing `ruleId` in favor of using a filter instead.

Reply:
Does this change mean that you could pass a constructed `filter` to this hook which will return something different than what the hook name and typing suggests?

Reply:
Yes, in the future we will use the new API that Xavier is working on to fetch this information. I will rename this hook to make it more clear and also more compatible with the future use-case 👍🏻